runelite-client/src/main/java/net/runelite/client/plugins/bankvalue/BankValuePlugin.java

```diff
@@ -54,45 +60,101 @@
 	private BankCalculation bankCalculation;
 
 	@Inject
-	private BankTitle bankTitle;
+	private BankValueConfig config;
+
+	@Inject
+	private BankSearch bankSearch;
 
 	@Provides
 	BankValueConfig getConfig(ConfigManager configManager)
 	{
 		return configManager.getConfig(BankValueConfig.class);
 	}
 
+	private int lastTab = -1;
+
 	@Override
 	protected void shutDown()
 	{
-		clientThread.invokeLater(bankTitle::reset);
+		clientThread.invokeLater(() -> bankSearch.reset(false));
```

Comment:
What's with the behavior change here?

Reply:
This just forces a rebuild of the bank, which resets the title since I'm doing it in script now.